ci/jenkins/test.sh

```diff
@@ -979,6 +998,15 @@ EOF
     done
 }
 
+# configure_kind_ipam_routes configure routes for non ipam test pods.
+function configure_kind_ipam_routes {
+    local kind_node="$1"
+
+    local pod_cidr=$(kubectl get nodes "${kind_node}" -o yaml | grep podCIDR | awk 'FNR == 1 {print$2}')
+    local node_ip=$(kubectl get nodes "${kind_node}" -o wide | awk 'FNR == 2 {print$6}')
+    sudo ip route add "$pod_cidr" via "$node_ip"
```

Comment:
I think it should do this in kind-setup.sh to concentrate testbed setup code.

There are already similar code in kind-setup.sh for other test cases.